### ISSUE TITLE
Add Greybird Openbox Theme

### ISSUE THREAD

**Issue by dark-yux** [shimmerproject/Greybird#92]
The addition of the theme would be a great addition for openbox users. http://box-look.org/content/show.php/?content=154075

**Comment by ochosi** [comment]
Right, it doesn't look like Greybird's window borders though. If the icons and the corner radius would match (and the author agrees) I wouldn't mind to include it.

**Comment by satya164** [comment]
@ochosi I don't think Openbox supports rounded corners.

**Comment by ochosi** [comment]
Right, then at least the icons and the gradient should match.

**Comment by dark-yux** [comment]
Thanks.

**Comment by satya164** [comment]
@dark-yux can you contact the author of this theme?

**Comment by snilek** [comment]
I use my own customised openbox theme, which is based on this one:
http://box-look.org/content/show.php/dumetella?content=147495
but modified to better fit the greybird look.
The base theme is free for customisation (do-whatever-you-want licence), and so is mine. If interested, please tell me the preferred way upload/send it.

**Comment by qwertychouskie** [comment]
Any updates on this?  I would love a proper Greybird theme for Openbox.

**Comment by fblais** [comment]
Ditto please!
